hatchery/src/world.rs

```diff
@@ -459,6 +487,31 @@ fn host_query(
         .expect("TODO: error handling")
 }
 
+fn host_host_query(
```

Comment:
not so sure about the host_host thing, is there not a clearer way of naming this?

Reply:
I don't like it much either. Maybe I can come up with another name besides "host".